script/BetterScript.s.sol

```diff
@@ -2,26 +2,36 @@
 pragma solidity ^0.8.23;
 
 import {Script, console2 as console} from "forge-std/Script.sol";
+import {Sphinx, Network} from "@sphinx-labs/plugins/SphinxPlugin.sol";
 
 interface ICREATE3Factory {
     function deploy(bytes32 salt, bytes memory creationCode) external payable returns (address deployed);
 
     function getDeployed(address deployer, bytes32 salt) external view returns (address deployed);
 }
 
-contract BetterScript is Script {
+contract BetterScript is Script, Sphinx {
     ICREATE3Factory internal factory = ICREATE3Factory(0x2Dfcc7415D89af828cbef005F0d072D8b3F23183);
 
-    modifier broadcast(uint256 key) {
-        vm.startBroadcast(key);
-        _;
-        vm.stopBroadcast();
-    }
-
-    modifier broadcastAddr(address who) {
-        vm.startBroadcast(who);
-        _;
-        vm.stopBroadcast();
+    function setUp() external {
+        sphinxConfig.owners = [address(0)]; // Add owner address(es)
+        sphinxConfig.orgId = ""; // Add org ID
+        sphinxConfig.testnets = [
+            Network.sepolia,
+            Network.arbitrum_sepolia,
+            Network.avalanche_fuji,
+            Network.linea_goerli,
```

Comment:
We should probably address the linea goer transaction bug before sending this over.

Reply:
Looks like you resolved that today, so I think we're good to go